README.md

```diff
@@ -1,3 +1,44 @@
 # eos-icons-adobe-xd
 
 An Adobe XD plugin to use all of EOS-icons from within the application
+
+## Steps to build it locally
+
+# Latest
+
+- Clone the repo in the folder of your choice.
+- Run `npm i` and `npm run watch`.
+- Install [`Adobe UXP Developer Tool`](https://www.adobe.io/photoshop/uxp/guides/uxp-developer-tool/) .
+- Open `Adobe XD` and open a document.
+- Open UXP Developer Tool.
+- Click `Add Existing Plugin`.
+- Add the `manifest.json` from the dist folder.
+- Click on 3 dot icon and select `Load`.
+- Go back to `Adobe XD` and try it out! :)
+
+# Deprecated
```

Comment:
I would remove this as it is no longer needed

Reply:
Removed.